### PR TITLE
chore(settings): improvements

### PR DESCRIPTION
**Pull request by banklesss**
👀

### CODE REVIEW

**Comment by banklesss** [comment]
fixing the margin issue

**Comment by michalszorad** [comment]
Hello @banklesss , is this still a draft of ready for a review?

**Comment by banklesss** [comment]
> Hello @banklesss , is this still a draft of ready for a review?

What I'm fixing isn't related to anything done here so feel free to suggest.

File: apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx [hunk 1]
```diff
@@ -49,102 +49,98 @@ export const ApplicationSettingsScreen = () => {
   }
 
   return (
-    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.settings}>
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx:52]: seems it isn't needed... do you know any edge case?

File: apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx [hunk 2]
```diff
@@ -49,102 +49,98 @@ export const ApplicationSettingsScreen = () => {
...
-      <ScrollView bounces={false}>
-        <StatusBar type="dark" />
-
-        <SettingsSection title={strings.general}>
-          <NavigatedSettingsItem
-            icon={<Icon.Globe {...iconProps} />}
-            label={strings.selectLanguage}
-            onNavigate={() => settingsNavigation.navigate('change-language')}
-            selected={language.label}
+    <ScrollView bounces={false} style={styles.settings}>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx]: @banklesss this is a screen it's required to be wrapped around a SafeAreaView with `top` `left` `right` edges.

File: apps/wallet-mobile/src/features/Settings/SettingsScreenNavigator.tsx [hunk 3]
```diff
@@ -136,7 +136,7 @@ const SettingsTabNavigator = () => {
     <Tab.Navigator
       screenOptions={({route}) => ({
         tabBarLabel: route.name === 'wallet-settings' ? strings.walletTabTitle : strings.appTabTitle,
-        tabBarStyle: {backgroundColor: COLORS.WHITE, elevation: 0, shadowOpacity: 0},
+        tabBarStyle: {backgroundColor: COLORS.WHITE, elevation: 0, shadowOpacity: 0, marginHorizontal: 16},
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/SettingsScreenNavigator.tsx:138]: best solution found

Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/SettingsScreenNavigator.tsx:138]: fixes scroll issue

File: apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx (hunk 2 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx]: `bottom` is automatically handle only when there is BottomTabBar, if there is not, then it is also required.

File: apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx [hunk 4]
```diff
@@ -185,6 +165,52 @@ const useLogout = () => {
...
+  }
+
+  const onDisableEasyConfirmation = () => {
+    navigation.navigate('app-root', {
+      screen: 'settings',
+      params: {
+        screen: 'disable-easy-confirmation',
+      },
+    })
+  }
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx]: extract to useNavigateTo -> .disableEasyConfirmation | .enableEasyConfirmation

**Review by stackchain — COMMENTED** [review]
minor CRs, lgtm

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx [hunk 5]
```diff
@@ -268,9 +259,12 @@ const messages = defineMessages({
 })
 
 const styles = StyleSheet.create({
+  root: {
+    height: '100%',
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx]: best solution found

File: apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx [hunk 6]
```diff
@@ -26,100 +28,82 @@ const iconProps = {
...
   const logout = useLogout()
   const settingsNavigation = useNavigation<SettingsRouteNavigation>()
+  const easyConfirmationEnabled = useEasyConfirmationEnabled(wallet)
 
   const onSwitchWallet = () => {
     resetToWalletSelection()
   }
 
-  const onEnableEasyConfirmation = () => {
-    navigation.navigate('app-root', {
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx:41]: now its going directly to the screen

Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx:41]: https://github.com/Emurgo/yoroi/pull/2676/files#diff-b0510212f2210c23d1f4777217f94d522d360f2d316cfa4070e5567d64249354R10

File: apps/wallet-mobile/src/features/Menu/Menu.tsx [hunk 7]
```diff
@@ -41,7 +41,7 @@ export const Menu = () => {
     <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.root}>
       <ScrollView contentContainerStyle={styles.scrollViewContent} bounces={false}>
         <AppSettings //
-          label={strings.appSettings}
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Menu/Menu.tsx:44]: product suggestion

File: apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx (hunk 5 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx]: found a better solution

**Review by michalszorad — APPROVED** [review]

**Pull request merged**